specs/pvs/corla.pvs

```diff
@@ -372,8 +405,8 @@ BEGIN
 % @todo Each scanner has a unique id number.
   vvpat: TYPE
   verified?: TYPE = pred[storage_container]
-  log: TYPE
-  chain_of_custody_log: TYPE FROM log
+
+  vvsg_log: TYPE
```

Comment:
Yes, I am talking about the kinds of logs defined in the VVSG.  I'll expand and explain.